### DISCUSSION TITLE
Benchmark against Biome

### DISCUSSION THREAD

**Discussion by ClementNerma** [oxc-project/oxc#6271] [General]
The recent article from Evan You on `void(0)` (see https://voidzero.dev/posts/announcing-voidzero-inc) states the following:

![image](https://github.com/user-attachments/assets/0aec8e70-3300-4f57-b59a-e076e951d96f)

But I haven't been able to find any benchmark against Biome. Stating that `oxlint` is **"the fastest"** requires to show actual benchmark, otherwise it's meaningless.

Maybe I've just not researched properly and there is a benchmark suite somewhere? In which case, please point it out to me :)

Otherwise, it would be interesting to create a benchmark to compare it against Biome to check if Evan You's statements is true or not.

**Comment by Brooooooklyn** [comment]
From the biome repo **_Biome's ~15x times faster than ESLint (without any plugins)_** https://github.com/biomejs/biome/tree/main/benchmark#linting

**Comment by Boshen** [comment]
https://github.com/oxc-project/bench-javascript-linter/issues/8 I'll restrict the rules to a simple set to make things comparable.
